Header header
string object_class
int8 object_ID
float64 confidence 
float64[] bounding_box_vertices  
float64[] bounding_box_center 
float64[] bounding_box_vertices_meter
float64[] bounding_box_center_meter
float64[] bounding_box_wh
int8[] segmentation_mask
